### ISSUE TITLE
Problem with my python? 

### ISSUE THREAD

**Issue by Wordprestesterino** [DeFiMasterd/NFTX-Auto-Minting-Bot#2]
This is the  error I'm experiencing. I'm running python 3.6.5. When I use the original settings file, the bot seems to work.

Traceback (most recent call last):
  File "minter.py", line 36, in <module>
    e = l(sc)
  File "C:\Users\x\AppData\Local\Programs\Python\Python36\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "C:\Users\x\AppData\Local\Programs\Python\Python36\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\x\AppData\Local\Programs\Python\Python36\lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 2 column 3 (char 4)

![lakka](https://user-images.githubusercontent.com/95507788/217600872-f0fec657-0301-4383-8f90-ecb738c0f0a2.PNG)

Any idea? Thanks in advance! Love your work!

**Comment by DeFiMasterd** [comment]
Will check it out and let you know!

**Comment by Wordprestesterino** [comment]
> Will check it out and let you know!

Thanks!

**Comment by DeFiMasterd** [comment]
So you filled your settings file with the specific data and you got this error correct?

This seems like you deleted the double quotes around the variables. If there is a line for example Wallet Address = "abc" you need to fill your details between the double quotes, not delete them

**Comment by Wordprestesterino** [comment]
> So you filled your settings file with the specific data and you got this error correct?
> 
> This seems like you deleted the double quotes around the variables. If there is a line for example Wallet Address = "abc" you need to fill your details between the double quotes, not delete them

Hey bro! In the attachment you can see my settings.json file. Is this the correct way? 
![github reply](https://user-images.githubusercontent.com/95507788/218254591-8dd6f71b-e217-45df-accd-75a7f66eb6af.PNG)

**Comment by Wordprestesterino** [comment]
This is what my json file looks like, but still doesn't seem to be working unfortunately. Should I Fill in every setting?

**Comment by DeFiMasterd** [comment]
Yes that should work like a charm, I just ran the code and it worked right away. Can you install python 3.8 and try it again?

**Comment by Wordprestesterino** [comment]
> Yes that should work like a charm, I just ran the code and it worked right away. Can you install python 3.8 and try it again?

Oh okay I see, however I'm not familiar with the possible values for; price_method or minting_method. Are these values I can  find in the contract adress? Does it have a specific name? I tried to google it, but unfortunately haven't found an answer yet. The contract adress I'm using for example is; 0x7f29b85834d6A2ba4BB1c64325686C6057B1B3C5 .

**Comment by DeFiMasterd** [comment]
So just fill the contract address, wallet and key, and then the bot should start without any errors, can you try

**Comment by Wordprestesterino** [comment]
> So just fill the contract address, wallet and key, and then the bot should start without any errors, can you try

Yes, I tried that. I still get the same error unfortunately.

**Comment by Wordprestesterino** [comment]
I made a video showing my settings etc.

https://user-images.githubusercontent.com/95507788/218276853-3503f24e-91cf-4a25-8fa5-0796884fccb0.mp4

**Comment by DeFiMasterd** [comment]
I deleted the comment as you didn't hide your private key. But I saw the video, I will check it, there seems to be something wrong in the set up

**Comment by Wordprestesterino** [comment]
> I deleted the comment as you didn't hide your private key. But I saw the video, I will check it, there seems to be something wrong in the set up

no worries about the private key , i made a new metamask account, but thanks for the integrity! Thanks for all the help!

**Comment by Wordprestesterino** [comment]
btw have you been able to replicate the issue yet? if you weren't it might have been an issue with my python, so I could maybe install more add-ons?

**Comment by Wordprestesterino** [comment]
> I deleted the comment as you didn't hide your private key. But I saw the video, I will check it, there seems to be something wrong in the set up

Hey there! Unfortunately I still haven't been able to make it work. I want to try filling in the information for name_of_price_method and name_of_minting_method , as that maybe causes something. Do you perhaps have an article/video or whatever so I could find these 2 variables in a desired contract adress? Thanks in advance!

**Comment by Blockys69** [comment]
I spoke with him and he is working on a GUI version of the minting bot so I think wait for an update this week possibly :)

**Comment by Wordprestesterino** [comment]
> I spoke with him and he is working on a GUI version of the minting bot so I think wait for an update this week possibly :)

Oh great to hear! Thanks for the info!

**Comment by DeFiMasterd** [comment]
Hey! yes I am working on a GUI version because this code version has too much room for error, stay tuned! :D
